apps/slackbot/utilities/constants.py

```
@@ -279,3 +280,10 @@
     int(org_id) for org_id in os.environ.get("ACHIEVEMENTS_ALPHA_TESTING_ORG_IDS", "").split(",") if org_id
 ]
 ACHIEVEMENT_AWARD_HOUR_CST = int(os.environ.get("ACHIEVEMENT_AWARD_HOUR_CST", 9))
+
+class AlertLevel(StrEnum):
+    DEFAULT = "default"
+    INFO = "info"
+    SUCCESS = "success"
+    WARNING = "warning"
+    ERROR = "error"
```

Ruff (0.15.21)

[warning] No newline at end of file

Add trailing newline

(W292)